### PR TITLE
Reorganize Navigation/Create Getting Started

### PR DESCRIPTION
**Pull request by jasommerset**
Reorganizing navigation to include a getting started section, created landing pages and updated user guide landing page.

### CODE REVIEW

**Review by sdconrox — CHANGES_REQUESTED** [review]
Looks like there are some pages moving so we should think about if that's necessary and if so, update the links on other pages.

File: source/getting_started/terminology.rst
[the remark this answers is not in the document]
Comment (jasommerset) [reply source/getting_started/terminology.rst:1]: Moved back, will address in new PR

File: source/getting_started/installing.rst
[the remark this answers is not in the document]
Comment (jasommerset) [reply source/getting_started/installing.rst:1]: I updated all user guide pages referencing it

File: source/getting_started/datasets.rst
[the remark this answers is not in the document]
Comment (jasommerset) [reply source/getting_started/datasets.rst:1]: moved back to user_guides to avoid breaking links, will address in new PR

**Review by sdconrox — APPROVED** [review]

**Review by sdconrox — APPROVED** [review]

**Pull request merged**